core/src/com/unciv/models/translations/Translations.kt

```diff
@@ -262,7 +262,7 @@ fun String.tr(): String {
     }.toHashSet()
     val language = UncivGame.Current.settings.language
 
-    if (contains('<')) { // Conditionals!
+    if (contains('<') && contains('>')) { // Conditionals!
```

Comment:
Technically this doesn't check the order of these, so it could still be that `.removeConditionals()` returns the same string, leading to infinite recursion of `.tr()` calls on the next line. But this is a minor point, as this PR fixes the main way this can happen anyway